ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/part/widgets/editortab/EditorTabWidget.java

```diff
@@ -186,12 +185,6 @@ public void unSelect() {
 
     /** {@inheritDoc} */
     @Override
-    public void setTabPosition(@NotNull TabPosition tabPosition) {
```

Comment:
>1. The method was newer used.

If it is not used it is not the reason to remove it. Deprecate it first. We have agreement about deprecation: https://github.com/eclipse/che/wiki/Development-Process#deprecation-policy read carefully.